inky/types.go

```diff
@@ -4,7 +4,7 @@
 
 package inky
 
-//go:generate go install golang.org/x/tools/cmd/stringer@latest
+//go:generate -command stringer go run golang.org/x/tools/cmd/stringer@latest
 //go:generate stringer -type=Model,Color,ImpressionColor -output types_string.go
```

Comment:
Thanks, #til! Did you mean to merge the two lines?

Reply:
I think that keeping the two lines allows to avoid a very long line where stringer arguments would be on the far right. This helps readability.

Reply:
@maruel ^^^ Do you still want me to merge the two lines?

Reply:
Ah ok I understand now.